packages/app/src/styles/misc.ts

```diff
@@ -64,6 +64,10 @@ export const misc = css`
         color: var(--color-negative);
     }
 
+    .tag.ellipsis {
+        max-width: 100px;
+    }
+
```

Comment:
I think this rule is much too broad. Can we add this [here](https://github.com/padloc/padloc/blob/main/packages/app/src/elements/items-list.ts#L163) instead? Also, please try to use `em` as sizing units wherever possible.

Reply:
Done.